### PR TITLE
Data creator with extended functionality

### PR DESCRIPTION
**Pull request by DaanvanderValk**

### CODE REVIEW

File: demo/dataCreator.js [hunk 1]
```diff
@@ -22,139 +17,277 @@ var DataCreator = function() {
     var minIntendedReviewTime = 5;
     var maxIntendedReviewTime = 35;
 
-    var currentUserName = "DaanvanderValk";
-
-    var currentDateTime = new Date(timeStart);
+    var currentUserName, currentDateTime;
 
     // This element types are used to create some random scroll data
```
Comment (borek2) [inline demo/dataCreator.js]: A sentence should end with a period.

File: demo/dataCreator.js [hunk 2]
```diff
@@ -22,139 +17,277 @@ var DataCreator = function() {
...
     var elementTypes = [101, 102, 103,104,105,106,107,108,109,110,111,112,113,114,115,201,202,203,301,302,303,304,305,401,402,403,404,405,501,502];
 
     // This needs be a hardcoded array, because we need real data to link to GitHub
```
Comment (borek2) [inline demo/dataCreator.js]: A sentence should end with a period.

File: demo/dataCreator.js [hunk 3]
```diff
@@ -22,139 +17,277 @@ var DataCreator = function() {
...
+            }
+        },
     ];
 
-    this.run = function() {
-        var compareTimeEnd = Number(new Date(timeEnd));
+    // Array of booleans, indicate whether the simulation is still open
+    var pullRequestsStillOpen = Array(pullRequests.length).fill(true);
+
+    this.run = function(inUser, inStart, inEnd) {
```
Comment (borek2) [inline demo/dataCreator.js]: You forgot a jdoc comment

File: demo/dataCreator.js [hunk 4]
```diff
@@ -22,139 +17,277 @@ var DataCreator = function() {
...
+        currentUserName = inUser;
+        currentDateTime = new Date(inStart);
+        var compareTimeEnd = Number(new Date(inEnd));
+        console.log("Started generator at " + currentDateTime);
```
Comment (borek2) [inline demo/dataCreator.js]: ok i stop :D

File: demo/dataCreator.js [hunk 5]
```diff
@@ -22,139 +17,277 @@ var DataCreator = function() {
...
+                // Wait a random time between two pull requests
+                waitRandomMinutes(1,240);
+            } else {
+                waitRandomMinutes(1,10);
+            }
 
-            // Wait a random time between two pull requests
-            waitRandomMinutes(2,180);
         }
+        console.log("Finished generator at " + currentDateTime);
```
Comment (lvdoorn) [inline demo/dataCreator.js]: Console.log! :police_car:

**Pull request merged**